tasks/databases-docker.yml

```diff
@@ -0,0 +1,93 @@
+# file: postgresql/tasks/databases-docker.yml
+
+- name: PostgreSQL | Make sure the PostgreSQL databases are present
+  postgresql_db:
+    name: "{{item.name}}"
+    owner: "{{ item.owner | default(postgresql_database_owner) }}"
+    encoding: "{{ item.encoding | default(postgresql_encoding) }}"
+    lc_collate: "{{ item.lc_collate | default(postgresql_locale) }}"
+    lc_ctype: "{{ item.lc_ctype | default(postgresql_ctype) }}"
+    port: "{{postgresql_port}}"
+    template: "template0"
+    state: present
+    login_user: "{{postgresql_admin_user}}"
+    login_password: "{{postgresql_admin_pwd}}"
+    login_host: "{{postgresql_login_host}}"
+  with_items: "{{postgresql_databases}}"
+  when: postgresql_databases|length > 0
+
+- name: PostgreSQL | Add extensions to the databases
+  shell: "psql {{item.0.db}} --username {{postgresql_admin_user}} --host {{postgresql_login_host}} --port {{postgresql_port}} -c 'CREATE EXTENSION IF NOT EXISTS {{ item.1 }};'"
+  become: yes
+  with_subelements:
+    - "{{postgresql_database_extensions}}"
+    - extensions
+  register: result
+  changed_when: "'NOTICE' not in result.stderr"
+  environment:
+    PGPASSWORD: "{{postgresql_admin_pwd}}"
+
+- name: PostgreSQL | Add hstore to the databases with the requirement
+  become: yes
+  shell: "{{ postgresql_bin_directory}}/psql {{item.name}} --username {{postgresql_admin_user}} --host {{postgresql_login_host}} --port {{postgresql_port}} -c 'CREATE EXTENSION IF NOT EXISTS hstore;'"
```

Comment:
Even with the docker support, that setting always refers to the directory on the host where the playbook runs (not where the postgres process runs). So, in case of errors, the variable can be changed without unwanted side-effects.

`psql` is indeed a requirement for the ansible host, so I will add a way to install it (just the client) in docker mode.